### PR TITLE
FIX screen size fall

### PR DESCRIPTION
**Pull request by john-botella**
# Fix 
On medium screens, left and right tables ride themselves and add difficulties to click on slice discounts button

### CODE REVIEW

File: htdocs/theme/eldy/style.css.php [hunk 1]
```diff
@@ -1357,6 +1357,23 @@
 	}
 }
 
+/* Force values on one colum for small screen */
```
Comment (eldy) [inline htdocs/theme/eldy/style.css.php:1360]: The 'fichehalfleft' and 'fichehalfright' are already 2 class to automatically switch on 1 column where scrren is too small.
image

The div <div class="div-table-responsive-no-min"> was missing. I added it.
Is it enough now on your side.

**Comment by john-botella** [comment]
Yes for small size like smart phone but not for small size like PC screen < 1600px

I introduce fichehalfleft-lg and fichehalright-lg to have an other break point at 1600px

I think i will introduce in develop : xxxxx-md , xxxxx-sm to have tablettes devices and PC small screen responsive support.


note : md -> medium
sm -> small

**Comment by eldy** [comment]
There is only 2 possibilities for 2 column: right and left or on top and under. 
And we should be able to switch in a mode or another by using CSS to change this according to size of screen. So adding more css style here will create a duplicate of css class.
If we want one columns for 1600px, we should instead modify the css for the existing style the is dedicated for that but currently, theme suppose that with 1600px, we have enough room to have 2 columns and I tested and i don't experience any trouble now the closing div has been fixed to have 2 columns with 1600 px.
Can you provide a screenshot you have now the /div bug has been fixed ?

**Comment by eldy** [comment]
Moved into develop branch

**Pull request merged**